astroplan/tests/test_core.py

```diff
@@ -155,3 +152,591 @@ def test_FixedTarget_ra_dec():
                                                         'SkyCoord')
     assert vega.coord.dec == vega_coords.dec == vega.dec, ('Retrieve Dec from '
                                                            'SkyCoord')
+
+def test_sunrise_sunset_equator():
+    '''
+    Check that time of sunrise/set for an observer on the equator is
+    consistent with PyEphem results (for no atmosphere/pressure=0)
```

Comment:
Perhaps you should also use some other comparison datapoint like the USNO sunrise/sunset calculator?  PyEphem shouldn't really be taken as any more "authoritative" than anywhere else in this respect, so comparing a few different things might be useful.